corehq/apps/es/es_query.py

```diff
@@ -71,6 +71,20 @@
 These things should nearly always be excluded, but if necessary, you can remove
 these with ``remove_default_filters``.
 
+Running against production
+--------------------------
+Since the ESQuery library is read-only, it's mostly safe to run against
```

Comment:
it's safe in that you won't be doing any writes, but you could in theory do some horribly large inefficient query which slows things down.